pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tamr-unify-client"
-version = "0.14.0-dev"
+version = "0.0.0"
```

Comment:
How does this effect the version of `tamr-unify-client` going forward?  Is this setting the version number to `0.0.0`?

Reply:
In dev, the version will now always be `0.0.0`. CI will automatically [replace with the the correct version](https://github.com/Datatamer/tamr-client/pull/476/files#diff-7c44ab2f659232c7a7dfa2847740c40e531b9f1c04a9d316e715b336323bc9c4R11) during the release process.